### ISSUE TITLE
Accessibility: FeatureCard Contrast & Focus Indicator

### ISSUE THREAD

**Issue by jaydonkrooss** [tl-its-umich-edu/canvas-course-manager-next#703]
Problematic component: `FeatureCard.tsx`

 Issue summary: 
- Dashboard insufficient contrast: tile background + text color and tile focus indicator

Based on these reported problems found in [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

- [Severity: Critical] CCM dashboard
    - The grey text on the grey background used for the tiles does not provide sufficient contrast (3.9:1; should be at least 4.5:1)
- [Severity: Critical] CCM dashboard
    - Focus is indicated by making the tile darker; this may not be perceived and provides even less contrast than the tile text/background noted above
